### ISSUE TITLE
resize (or even hide) Test Coverage panel

### ISSUE THREAD

**Issue by DinisCruz** [jacogr/atom-lcov-info#12]
When coding in TDD style I don't need to see all that info all the time

![image](https://cloud.githubusercontent.com/assets/656739/5681804/7a85b6f8-9813-11e4-8346-08e8c116d0f6.png)

In fact, is there a way to just open that info in a separate tab?

**Comment by DinisCruz** [comment]
one way of doing it is to add the **display:none;**  to the **.lcov-info-panel** style , which is inside the **lcov-info-.less** file

![image](https://cloud.githubusercontent.com/assets/656739/5682896/6bf36740-981b-11e4-95e1-2911bce234ea.png)

which seems to do the trick

![image](https://cloud.githubusercontent.com/assets/656739/5682928/a1e401d4-981b-11e4-9e1a-0f27c5354ae3.png)

**Comment by jacogr** [comment]
Resizing is probably a bit more complex than it should be - could have the option to show it 

(a) docked
(b) in seperate tab
(c) never

**Comment by DinisCruz** [comment]
yes, with option b) being a good option , specially if you don't do it once automatically (and if the user closes it, then he/she needs to use a menu option to re-open it)

**Comment by DinisCruz** [comment]
Using the 

![image](https://cloud.githubusercontent.com/assets/656739/5706090/e29a8ad4-9a75-11e4-90e9-e68f60bcdd74.png)

I was able to use these values to control the visibility (and size) of the bottom pane

![image](https://cloud.githubusercontent.com/assets/656739/5706103/f7800cee-9a75-11e4-8bab-0c9d0c42c520.png)

Note that these only seem to work if the css value set in the **styles.less** do not exist on the main css **lcov-info.less** file. Which means that to make it work I had to make this small change to **lcov-info.less**:

![image](https://cloud.githubusercontent.com/assets/656739/5706122/21251e54-9a76-11e4-995b-57f7e41121c3.png)

**Comment by DinisCruz** [comment]
Btw, here is a good example of how I'm using this package (namely to get semi-realtime feedback on the code coverage)

https://github.com/TeamMentor/TM_4_0_Design/wiki/Coffee-UnitTests-tricks
